src/content/onlineservice/sopcast_content_handler.cc

```diff
@@ -144,18 +144,18 @@ std::shared_ptr<CdsObject> SopCastContentHandler::getObject(const std::string& g
     // I wish they had a mimetype setting
     //auto mappings = config->getDictionaryOption(CFG_IMPORT_MAPPINGS_EXTENSION_TO_MIMETYPE_LIST);
```

Comment:
I suggest to create a new config dictionary `CFG_IMPORT_SOPCAST_MIMETYE_LIST` defaulting to the current values and some string with the default value.

Reply:
That's beyond the scope here.